netbox/dcim/api/serializers.py

```diff
@@ -171,6 +171,16 @@ class Meta:
         fields = ['id', 'rack', 'units', 'created', 'user', 'tenant', 'description']
 
 
+class RackElevationDetailFilterSerializer(serializers.Serializer):
+    face = serializers.ChoiceField(choices=['front', 'rear'], default='front')
```

Comment:
`choices` and `default` should reference `dcim.choices.DeviceFaceChoices`. We should never be setting static choice values directly on fields.